### PR TITLE
MBS-8293 Add groovy samples for InstrumentationTestsPlugin

### PR DESCRIPTION
**Pull request by sergioboi**

### CODE REVIEW

File: docs/content/docs/test/Runner.md [hunk 1]
```diff
@@ -17,8 +17,11 @@ It uses an internal TMS (test management system). We are working on support othe
 
 ## Getting started
 
-1. Apply the instrumentation-tests gradle plugin
-Add to your build.gradle.kts
+1. Apply the instrumentation-tests gradle plugin \
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
1. Apply the `instrumentation-tests` Gradle plugin \
```

File: docs/content/docs/test/Runner.md [hunk 2]
```diff
@@ -17,8 +17,11 @@ It uses an internal TMS (test management system). We are working on support othe
...
+Add to your `build.gradle.kts` or `build.gradle`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:21]: I just wondering how Gradle describes it in the documentation. They include a file name in code blocks.
https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block

My proposal: move a file name to a tab description:
- Remove the line: "Add to your `build.gradle.kts` or `build.gradle`". It's obvious from samples.
- `{{< tab "build.gradle.kts" >}}`

File: docs/content/docs/test/Runner.md [hunk 3]
```diff
@@ -293,14 +576,48 @@ configurationsContainer.register("Local") {
...
+                api27 {
+                    deviceName = "API27"
+                    scheduling {
+                        quota {
+                            retryCount = 1
+                            minimumSuccessCount = 1
+                        }
+
+                        testsCountBasedReservation {
+                            device = new LocalEmulator("27", 27, "Android_SDK_built_for_x86")
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:601]: Can we use a factory method here?

File: docs/content/docs/test/Runner.md (hunk 2 above)
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]: I just wondering how Gradle describes it in the documentation. They include a file name in code blocks.
https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block

My proposal: move a file name to a tab description:
- Remove the line: "Add to your `build.gradle.kts` or `build.gradle`". It's obvious from samples.
- `{{< tab "build.gradle.kts" >}}`

Comment (sergioboi) [reply docs/content/docs/test/Runner.md]: Good idea

File: docs/content/docs/test/Runner.md (hunk 3 above)
Comment (sergioboi) [reply docs/content/docs/test/Runner.md:601]: I think after infra release we can. But I need to change build.gradle in Avito and then documentation.
Consider it's not a blocker.

File: docs/content/docs/test/Runner.md (hunk 2 above)
Comment (sergioboi) [reply docs/content/docs/test/Runner.md]: For sameness, I left Kotlin and Groovy tabs, but deleted lines with `build.gradle.kts`, `build.gradle` where could and moved `build.gradle.kts` and `build.gradle` into tabs.

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**